packages/client/src/client/open-feature-client.ts

```diff
@@ -179,7 +179,7 @@ export class OpenFeatureClient implements Client {
     const allHooksReversed = [...allHooks].reverse();
 
     const context = {
-      ...OpenFeature.getContext(),
+      ...OpenFeature.getContext(this?.options?.name),
```

Comment:
We simply should have passed the context name here, if set, and forgot.

Reply:
We should probably add a test to make sure the correct context is being sent to hooks and to the resolve call. I missed that cause when adding client side context management feature.

Reply:
Nice catch and done.